vault/router_test.go

```diff
@@ -348,6 +348,10 @@ func TestRouter_LoginPath(t *testing.T) {
 		Login: []string{
 			"login",
 			"oauth/*",
+			"end1/+",
+			"end2/+/",
+			"end3/+/*",
+			"+/begin",
```

Comment:
Can we also add a case for more than one wildcard path, e.g. `end4/+/+/login`?

Reply:
Good idea!